File: backoffice/src/restaurant/domain/valueObjects/RestaurantId.py
"""
 *
 * Libraries 
 *
"""

from src.shared.domain import UuidValueObject

"""
 *
 * Class
 *
"""

class RestaurantId( UuidValueObject ):

    """
     *
     * Methods 
     *
    """
    
    def __init__( self, value : str = None ) -> None:
        super().__init__( value )

File: preparation/src/restaurant/domain/RestaurantId.cs
/* 
 * 
 * Libraries
 *
*/

using preparation.src.shared.domain;

/* 
 * 
 * Class
 *
*/

namespace preparation.src.restaurant.domain {

    public class RestaurantId : UuidValueObject {

        /* 
         * 
         * Methods
         *
        */

        public RestaurantId( string value ) : base( value ) {
        }

        public RestaurantId() {
        }

    }

}